### ISSUE TITLE
inlet profile from file is not initialized from config.cfg

### ISSUE THREAD

**Issue by bigfooted** [su2code/SU2#1265]
When SU2 creates an inlet profile file, the values in the file are set to 0.0, expect for the vertex coordinates. The values used to be initialized from MARKER_INLET in the config file. Any particular reason this initialization disappeared? I propose to add this again, and also to add a line to inlet.dat with the names of the variables in the columns so it's easier for users to see what's in the columns.

**Comment by bigfooted** [comment]
implementation result:

```
NMARK= 2
MARKER_TAG= inlet_air
NROW=32
NCOL=8
# COORD-X               COORD-Y                 TEMPERATURE             VELOCITY                NORMAL-X                NORMAL-Y                TKE                     DISSIPATION
-2.500000000000000e-01  5.000000000000000e-02   3.000000000000000e+02   5.000000000000000e+00   1.000000000000000e+00   0.000000000000000e+00   1.500000000000000e-02   1.000000000000000e+02
-2.500000000000000e-01  4.838709677419802e-02   3.000000000000000e+02   5.000000000000000e+00   1.000000000000000e+00   0.000000000000000e+00   1.500000000000000e-02   1.000000000000000e+02


```

**Comment by pcarruscag** [comment]
Sounds good so long as backwards compatibility is kept and the names for the columns and default values are passed into CMarkerProfileReaderFVM in a way that does not require it to do anything problem-specific.

**Comment by bigfooted** [comment]
yes, it works with and without the additional column information.

**Comment by bigfooted** [comment]
closed by #1268
